applications/StructuralMechanicsApplication/custom_constitutive/yield_surfaces/modified_mohr_coulomb_yield_surface.h

```diff
@@ -291,8 +291,8 @@ class ModifiedMohrCoulombYieldSurface
         else
             c1 = 0.0; // check
 
-        if (checker < 29.0) {
-            c2 = cos_theta * CFL * (K1 * (1 + tan_theta * tan_3theta) + K2 * sin_phi * (tan_3theta - tan_theta) / Root3);
+        if (std::abs(checker) < 29.0) { // Lode angle not greater than pi/6
```

Comment:
Exactly! when the LodeAngle is close to pi/6 we have problems due to the edge... I think that is better to exclude de viccinity of the 30º because we are going to have instabilities, this is why we take 29º.

Reply:
29.9?, I don't know